websites/L/LCPDFR/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "1133734950055714866",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let urlParameters: URLSearchParams, searchQuery: string;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/XAgWtZw.png",
+			startTimestamp: browsingTimestamp,
+		},
+		privacy = await presence.getSetting<boolean>("privacy"),
+		{ search, pathname, hostname } = document.location;
+
+	switch (hostname) {
+		case "www.lcpdfr.com": {
+			if (pathname === "/") presenceData.details = "Patrolling the Home Page";
+			else if (pathname.includes("/downloads/")) {
+				const path = pathname.split("/");
+				presenceData.smallImageKey = Assets.Viewing;
+				if (path.includes("essential-mods"))
+					presenceData.details = "Inspecting Essential Mods";
+				else if (path.includes("vehiclemodels"))
+					presenceData.details = "Analyzing Vehicle Models";
+				else if (path.includes("vehiclestextures"))
+					presenceData.details = "Examining Vehicle Textures";
+				else if (path.includes("scripts"))
+					presenceData.details = "Investigating Scripts & Plugins";
+				else if (path.includes("character"))
+					presenceData.details = "Reviewing Character Mods";
+				else if (path.includes("audio"))
+					presenceData.details = "Exploring Audio Mods";
+				else if (path.includes("datafile"))
+					presenceData.details = "Examining Data Files";
+				else if (path.includes("misc"))
+					presenceData.details = "Keeping an eye out for miscelaneous";
```

Comment:
Consider implementing the following code:
```ts
document.querySelectorAll('[class="ipsTabs_item ipsTabs_activeItem"]')[1]?.textContent?.trim()
```
This gets the current tab used in the download part.
![image](https://github.com/PreMiD/Presences/assets/42322979/89da1937-495a-4f93-b0f1-e9d5eebf25e1)
![image](https://github.com/PreMiD/Presences/assets/42322979/b7ad31c6-8882-4f51-a300-21f3165214b4)

Reply:
I resolved this by adding the following line

`activeTab = document.querySelectorAll('[class="ipsTabs_item ipsTabs_activeItem ipsTabs_active"]')[0]?.textContent?.trim();`

I had to add 'ipsTabs_active' to ensure that it will get captured everytime. The one without it only worked once on the page but after going to the next tab it returned 'undefined'

Reply:
Perhaps the `ipsTabs_active` may not be needed if you use the dot class notation instead: `.ipsTabs_item.ipsTabs_activeItem`

Reply:
didn't think of that while writing it! That did work thanks, implemented it and pushed it to the PR